src/transformers/pipelines/token_classification.py

```diff
@@ -228,6 +234,70 @@ def __call__(self, inputs: Union[str, List[str]], **kwargs):
             return answers[0]
         return answers
 
+    def set_subwords_label(self, entities: List[dict]) -> List[dict]:
+        strategy = self.subword_label_re_alignment
+
+        def set_labels(sub_words: List[dict]) -> dict:
+            score = np.stack([sub["score"] for sub in sub_words])
+            if strategy == "default" or strategy is True:
```

Comment:
Using the enum would make things clearer here as we could do `AggregationStrategy.DEFAULT`